WcaOnRails/app/jobs/timed_application_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class TimedApplicationJob < ApplicationJob
```

Comment:
What about moving the *deferring* logic into `ApplicationJob`? I think it is useful itself actually.